skyrim-platform/src/platform_se/skyrim_platform/TextApi.cpp

```diff
@@ -60,6 +61,73 @@ JsValue TextApi::SetTextColor_(const JsFunctionArguments& args)
   return JsValue::Undefined();
 }
 
+JsValue TextApi::SetTextSize(const JsFunctionArguments& args)
+{
+  auto textId = static_cast<int>(args[1]);
+
+  auto size = static_cast<double>(args[2]);
+
+  TextsCollection::GetSingleton().SetTextSize(textId, size);
+  return JsValue::Undefined();
+}
+
+JsValue TextApi::SetTextRotation(const JsFunctionArguments& args)
+{
+  auto textId = static_cast<int>(args[1]);
+
+  auto rot = static_cast<double>(args[2]);
+
+  TextsCollection::GetSingleton().SetTextRotation(textId, rot);
+  return JsValue::Undefined();
+}
+
+JsValue TextApi::SetTextFont(const JsFunctionArguments& args)
+{
+  auto textId = static_cast<int>(args[1]);
+
+  std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+  auto argString = converter.from_bytes(static_cast<std::string>(args[2]));
+
+  std::wstring widestr = std::wstring(argString.begin(), argString.end());
```

Comment:
`argString` is already a properly converted UTF-16 string, no need to do anything else here

Reply:
fixed